cmd/hubble/subcmd_linux.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package hubble
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/spf13/cobra"
+	"go.etcd.io/etcd/version"
+)
+
+func Cmd(agentHive *hive.Hive) *cobra.Command {
+	hubbleCmd := &cobra.Command{
+		Use:   "hubble-control-plane",
```

Comment:
It cannot be a flag, it's very different than current control plane and requires all new flags and setup. IMO, a new sub command makes sense. And because it's. sub-command, I kept it a noun.